server/src/test/proto/spine/test/event/enrichment/enrichment.proto

```diff
@@ -43,3 +45,46 @@ message ProjectCreatedEnrichmentAnotherPackageFqnAndMsgOpt {
 
     string project_name = 1 [(by) = "spine.test.event.ProjectCreated.project_id"];
 }
+
+// The enrichment for all the events declared in the "spine.test.event.user" package.
+//
+// See `EventEnrichmentsMapShould#contain_all_events_from_package_by_one_enrichment` test for details.
+//
+message UserPackageEventsEnrichment {
+    option (enrichment_for) = "spine.test.event.user.*";
+
+    spine.time.ZoneOffset time_zone = 1 [(by) = "*.user_uid"];
+}
+
+// The enrichment for all the event declared in "spine.test.event.user.permissions" package and referencing user who
+// is able to grant permissions.
+//
+// See `EventEnrichmentsMapShould#contain_only_events_with_target_field_if_declared_though_package` test for details.
+//
+message GranterEventsEnrichment {
+    option (enrichment_for) = "spine.test.event.user.permissions.*";
+
+    spine.people.PersonName granter_name = 1 [(by) = "*.granter_uid"];
```

Comment:
There is no such a word as `granter`. The correct term is `grantor`, i.e. the person who grants.